content/05.discussion.md

```diff
@@ -1,8 +1,23 @@
-# Discussion
-Here mention why performnace increases in the beginning for the generative model then decreases
+## Discussion
 
-Discuss discriminator model performance given generative model
+We tested the feasibility of re-using label functions to extract relationships from literature.
+Through our sampling experiment we found that adding relevant label functions increases prediction performance (shown in the on-diagonals of Figures {@fig:generative_model_auroc} and {@fig:generative_model_auprc}).
+We found that label functions designed from relatively related edge types can increase performance (seen when GiG label functions predicts CbG and vise versa).
+We also found that one edge type (DaG) is agnositic to label function source (Figures {@fig:generative_model_auroc} and {@fig:generative_model_auprc}). 
+An interesting finding is performance increases when adding a single label funciton to the generative model baseline.
```

Comment:
funciton is misspelled. Has "generative model baseline" been explained elsewhere? It's not entirely clear here that these label functions are in addition to the knowledge base labeling functions.